kolibri/plugins/coach/assets/src/views/new/ReportsQuizHeader.vue

```diff
@@ -0,0 +1,97 @@
+<template>
+
+  <div>
+
+    <p>
+      <BackLink
+        :to="{ name:'NEW_COACH_PAGES', params: {page: 'ReportsLessonListPage'} }"
+        :text="$tr('back')"
+      />
+    </p>
+    <h1>Some quiz</h1>
+    <KDropdownMenu
+      slot="optionsDropdown"
+      :text="$tr('options')"
+      :options="actionOptions"
+      appearance="raised-button"
+      @select="goTo($event.value)"
+    />
+    <dl>
+      <dt>{{ $tr('status') }}</dt>
+      <dd><QuizActive :active="true" /></dd>
+      <dt>{{ $tr('recipients') }}</dt>
+      <dd><Recipients :groups="[]" /></dd>
+      <dt>{{ $tr('progress') }}</dt>
+      <dd><Completed :count="3" :total="10" :verbosity="1" /></dd>
+      <dt>{{ $tr('questionOrder') }}</dt>
+      <dd><QuestionOrder order="random" /></dd>
+    </dl>
+
+    <div>
+      <KButton
+        :text="$tr('learnerReport')"
+        appearance="flat-button"
+        @click="goTo('ReportsQuizLearnerListPage')"
+      />
+      <KButton
+        :text="$tr('difficulties')"
+        appearance="flat-button"
+        @click="goTo('ReportsQuizQuestionListPage')"
+      />
+    </div>
+
+    <hr>
+
+  </div>
+
+</template>
+
+
+<script>
+
+  import imports from './imports';
+
+  export default {
+    name: 'ReportsQuizHeader',
+    components: {},
+    mixins: [imports],
+    computed: {
+      actionOptions() {
+        return [
+          { label: this.$tr('preview'), value: 'ReportsQuizPreviewPage' },
+          { label: this.$tr('editDetails'), value: 'ReportsQuizEditorPage' },
+        ];
+      },
+    },
+    methods: {
+      goTo(page) {
+        this.$router.push({ name: 'NEW_COACH_PAGES', params: { page } });
+      },
+    },
+    $trs: {
+      back: 'Back to quizzes',
```

Comment:
This is WIP, so I guess the latest iteration with `View all quizzes` is still not in...

Reply:
thanks!